### ISSUE TITLE
git-appraise integration

### ISSUE THREAD

**Issue by archseer** [lambdalisue/gina.vim#169]
This is a long shot by far, but it would be neat if gina.vim had a [git-appraise](https://github.com/google/git-appraise) integration (show a diff, be able to tap a key on certain lines or selections to leave a comment, similar to blame mode possibly)

**Comment by lambdalisue** [comment]
Interesting.
Is the app associate with GitHub review system?

**Comment by archseer** [comment]
No, it's a separate system that doesn't need GitHub! It stores the code review inside git repository, so that you can use it anywhere, even your own server.

**Comment by lambdalisue** [comment]
Very interesting. I like that idea. However, I don't use that system and I don't have any plan to use that system.
I've found [git-pull-request-mirror](
https://github.com/google/git-pull-request-mirror) but I couldn't find any advantage for using that system to make a PR on GitHub instead of using GitHub review system directly (or am I missing some really useful use-case?) while I anyway use GitHub at least for now.
So I don't think **I** can implement that feature on gina.vim while I cannot imagine what kind of UI is useful, unfortunately.

But as I said I'm interesting, I'm very welcome to help people who are willing to make such feature on gina.vim 👍  Probably an external gina's plugin is the way to implement such one.
